### DISCUSSION TITLE
15.0 Stable

### DISCUSSION THREAD

**Discussion by KristopherMackowiak** [blakeblackshear/frigate#15427] [Ask A Question] [answered]
Is there an estimated release date/month for version 15? I'm using beta, so I wouldn't want to miss a release because I have an image attached instead of :stable :)

**Comment by hawkeye217** [comment]
No, there is no target release date. It's ready when it's ready. 

You could subscribe to release email notifications from GitHub so you don't miss the release.

**Comment by KristopherMackowiak** [comment]
Have you thought about building a lastest image alongside stable?

**Reply by hawkeye217** [reply, marked as answer]
Yes, dev builds are available here: https://github.com/blakeblackshear/frigate/pkgs/container/frigate
